modules/dnn/src/layers/prior_box_layer.cpp

```diff
@@ -180,21 +210,25 @@ class PriorBoxLayerImpl CV_FINAL : public PriorBoxLayer
     PriorBoxLayerImpl(const LayerParams &params)
     {
         setParamsFrom(params);
-        _minSize = getParameter<float>(params, "min_size", 0, false, 0);
         _flip = getParameter<bool>(params, "flip", 0, false, true);
         _clip = getParameter<bool>(params, "clip", 0, false, true);
         _bboxesNormalized = getParameter<bool>(params, "normalized_bbox", 0, false, true);
 
         _aspectRatios.clear();
 
+        _minSize.clear();
+        getMinSize(params);
```

Comment:
min_size should not be empty (otherwise exception `CV_Assert_N(!_boxWidths.empty(), !_boxHeights.empty());`). So let's add a check `!_minSize.empty()`.